src/content/docs/extensions/json.mdx

```diff
@@ -32,7 +32,7 @@ See our YouTube video for a walkthrough on how to use the JSON extension:
   </iframe>
 </div>
 
-## Example dataset
+### Example dataset
 
 Let's look at an example dataset to demonstrate how the JSON extension can be used.
 We have 3 JSON files that contain information about patients and their medical conditions. The
```

Comment:
`person.json` should be renamed to `patient.json` in this file.

Reply:
Ah thanks. Will fix soon.